packages/ifpack2/src/Ifpack2_BlockRelaxation_def.hpp

```diff
@@ -141,6 +141,9 @@ setParameters (const Teuchos::ParameterList& List)
     if (relaxType == "Jacobi") {
       PrecType_ = Ifpack2::Details::JACOBI;
     }
+    else if (relaxType == "MTJacobi") {
+      PrecType_ = Ifpack2::Details::MTJACOBI;
+    }
```

Comment:
@kyungjoo-kim, yes. I followed @mndevec's pattern. Here's the full list in Ifpack2_ConfigDefs.hpp:
```
   enum RelaxationType {
      JACOBI,  //!< Jacobi
      GS,      //!< Gauss-Seidel
      SGS,     //!< Symmetric Gauss-Seidel
      MTGS,    //!< Multicore Gauss-Seidel
      MTSGS,   //!< Multicore Symmetric Gauss-Seidel
      MTJACOBI //!< Multicore Jacobi
    };
```